### ISSUE TITLE
Grafana Datasource pointing to wrong service

### ISSUE THREAD

**Issue by vipinvkmenon** [cortexproject/cortex#6132]
**Describe the bug**
In the docs/getting-started, setting up of the grafana with the [grafana-dashboard](https://github.com/cortexproject/cortex/blob/4539330d0dbb0b72e168e9d3173a09e820296dbe/docs/getting-started/grafana-datasource.yaml#L8) is wrong. 

**To Reproduce**
1. Follow steps of the [tutorial](https://cortexmetrics.io/docs/getting-started/#instructions-1) to deploy the cortex and grafana
2. Open Grafana Dashboard

**Expected behavior**
Grafana should open cortex. I really it should be pointing to `http://cortex-nginx/api/prom 

**Environment:**
 - Infrastructure: Kubernetes
 - Deployment tool: helm

**Additional Context**

**Comment by yeya24** [comment]
> Grafana should open cortex. I really it should be pointing to `http://cortex-nginx/api/prom

What did you see instead? Is the address configured wrong?

**Comment by dsabsay** [comment]
There are two separate getting started demos: one with Docker Compose (single binary cortex) and another with Helm/Kind (microservices mode).

We may need to verify the grafanas are configured correctly for each one. This would be a good first issue:

1. Set up the docker-compose and verify Grafana is pointing to cortex
2. Set up the kind demo and verify Grafana is pointing to cortex

**Comment by Ajay-Satish-01** [comment]
@dsabsay can I work on this issue?

**Comment by dsabsay** [comment]
@Ajay-Satish-01 Yes! Thanks for volunteering. Please go ahead and let me know if you have any questions!

**Comment by Ajay-Satish-01** [comment]
@dsabsay please check if this is what you expected.
